### ISSUE TITLE
Demos stopped working

### ISSUE THREAD

**Issue by piotrjaw** [inspirit/jsfeat#84]
Hi, it seems that the demos stopped working at all, there seems to be a problem with the MediaStream:

<img width="436" alt="screenshot 2019-01-10 at 20 42 33" src="https://user-images.githubusercontent.com/1726537/50993280-8dddcd80-1519-11e9-9cc1-034fb20769f5.png">

<img width="492" alt="screenshot 2019-01-10 at 20 50 23" src="https://user-images.githubusercontent.com/1726537/50993300-99c98f80-1519-11e9-9540-8649106fdcc1.png">

**Comment by piotrjaw** [comment]
OK so it seems that calling `URL.createObjectURL` with a `MediaStream` is deprecated, but all you need to do is to replace:
```js
video.src = compatibility.URL.createObjectURL(stream);
```
with:
```js
video.srcObject = stream;
```
and it should work just fine. Would do it myself, but I don't see the demos sourcecode here :(

**Comment by donaldr** [comment]
Hard to know if I should use this library without seeing a demo. And yes, I realize I can download it and run it locally, but it would be so much easier if this simple change was made.

**Comment by lubosmato** [comment]
> OK so it seems that calling `URL.createObjectURL` with a `MediaStream` is deprecated, but all you need to do is to replace:
> 
> ```js
> video.src = compatibility.URL.createObjectURL(stream);
> ```
> 
> with:
> 
> ```js
> video.srcObject = stream;
> ```
> 
> and it should work just fine. Would do it myself, but I don't see the demos sourcecode here :(

Exactly. https://developer.mozilla.org/en-US/docs/Web/API/URL/createObjectURL#Usage_notes

> Important: If you still have code that relies on createObjectURL() to attach streams to media elements, you need to update your code to simply set srcObject to the MediaStream directly.
